src/main/java/de/tum/in/www1/artemis/service/connectors/BitbucketService.java

```diff
@@ -10,6 +10,7 @@
 import javax.annotation.Nullable;
 import javax.validation.constraints.NotNull;
 
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
Is there not a native Java exception for this?

Reply:
Not exactly, but I switched to `UnsupportedOperationException`, which means pretty much the same thing.